views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
+    <p>Where: {{event.location}}</p>
+    <p>With whom: {{event.attending}}</p>
+    <p>Why: {{event.image}}</p>
+  </div>
+  
+  <div>
+      <p>Do you want to partake in our revelry? Then RSVP!</p>
+      <form> 
+          <method="POST"></method>
+          <div>
+              <label for="email">E-mail:</label>
+              <input type="email" id="email" />
+          </div>
+          
+          <div class = "button">
```

Comment:
Thanks for this guidance! I removed all spaces before & after "=" and
removed all the <div></div>.

Francesco -- Where can I find your code for #24 in the Cloud9 interface?
I'd like to use it as a model, as David suggested.

On Mon, Feb 9, 2015 at 10:25 PM, David Hatch <notifications@github.com>
wrote:

> In views/event-detail.html
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25-23discussion-5Fr24386778&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=NozBVXD6MQW0Uv7p3WgdMMky1X0Z7KfPxa5IxhUmCvk&s=qpvGPQ_mM8NqIe9aVcZOh6eQexlcPYYF_BSdpEQmw-0&e=>
> :
>
> > +    <p>When: {{event.date}}</p>
> > +    <p>Where: {{event.location}}</p>
> > +    <p>With whom: {{event.attending}}</p>
> > +    <p>Why: {{event.image}}</p>
> > +  </div>
> > +
> > +  <div>
> > +      <p>Do you want to partake in our revelry? Then RSVP!</p>
> > +      <form>
> > +          <method="POST"></method>
> > +          <div>
> > +              <label for="email">E-mail:</label>
> > +              <input type="email" id="email" />
> > +          </div>
> > +
> > +          <div class = "button">
>
> No spaces before and after =. Curious as to why this div is needed at
> all. The submission should be accomplished via an input element of
> type='submit'. See the form @francescoannetta
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_francescoannetta&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=NozBVXD6MQW0Uv7p3WgdMMky1X0Z7KfPxa5IxhUmCvk&s=G1E9-HWbnYG2fDWIb2nUPS-83Vss1bAbzWRyOGgtTwg&e=>
> created in #24
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_24&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=NozBVXD6MQW0Uv7p3WgdMMky1X0Z7KfPxa5IxhUmCvk&s=ctc0XFXUfHT1Y5EDkpiH_BHkv96eQe5abW7-Q3jFTMU&e=>
> for an example.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25_files-23r24386778&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=NozBVXD6MQW0Uv7p3WgdMMky1X0Z7KfPxa5IxhUmCvk&s=GP0RwLuVhCW0fxHlQ4Db68CO3aLNlghMeYjlG-rbfTY&e=>
> .
>